libs/core/dtos/s3/s3-params-response.dto.ts

```diff
@@ -0,0 +1,42 @@
+/** DTO for s3 parameters response. */
+export interface S3ParamsResponseDto{
+
+	/** Policy. */
+	readonly 'policy': string;
+
+	/** Action status. */
+	readonly 'success_action_status': string;
```

Comment:
Quotes look redundant here, we can omit them:
```suggestion
	readonly policy: string;

	/** Action status. */
	readonly success_action_status: string;
```